diff --git a/src/Agenda.vala b/src/Agenda.vala
index 9bf5069..2e87711 100644
--- a/src/Agenda.vala
+++ b/src/Agenda.vala
@@ -40,6 +40,8 @@ namespace Agenda {
             }
 
             window = new AgendaWindow (this);
+            Gtk.Settings.get_default().set_property("gtk-theme-name", "elementary");
+            Gtk.Settings.get_default().set_property("gtk-icon-theme-name", "elementary");
             window.delete_event.connect (window.main_quit);
             window.show_all ();
             window.update ();
